*** chat.py.dst	2022-11-19 18:31:51.000000000 +0000
--- chat.py	2022-11-19 18:32:41.000000000 +0000
***************
*** 22,32 ****
  
  """Chat area."""
  
  import datetime
  
! from PySide6 import QtCore, QtWidgets, QtGui
  
  from qweechat import config
  from qweechat.weechat import color
  
  
--- 22,32 ----
  
  """Chat area."""
  
  import datetime
  
! from PyQt5 import QtCore, QtWidgets, QtGui
  
  from qweechat import config
  from qweechat.weechat import color
  
  
