*** debug.py.dst	2022-11-19 18:31:51.000000000 +0000
--- debug.py	2022-11-19 18:32:41.000000000 +0000
***************
*** 20,30 ****
  # along with QWeeChat.  If not, see <http://www.gnu.org/licenses/>.
  #
  
  """Debug window."""
  
! from PySide6 import QtWidgets
  
  from qweechat.chat import ChatTextEdit
  from qweechat.input import InputLineEdit
  
  
--- 20,30 ----
  # along with QWeeChat.  If not, see <http://www.gnu.org/licenses/>.
  #
  
  """Debug window."""
  
! from PyQt5 import QtWidgets
  
  from qweechat.chat import ChatTextEdit
  from qweechat.input import InputLineEdit
  
  
