*** qweechat.py.dst	2022-11-19 18:31:51.000000000 +0000
--- qweechat.py	2022-11-19 18:32:41.000000000 +0000
***************
*** 35,45 ****
  
  import sys
  import traceback
  from pkg_resources import resource_filename
  
! from PySide6 import QtCore, QtGui, QtWidgets
  
  from qweechat import config
  from qweechat.about import AboutDialog
  from qweechat.buffer import BufferListWidget, Buffer
  from qweechat.connection import ConnectionDialog
--- 35,45 ----
  
  import sys
  import traceback
  from pkg_resources import resource_filename
  
! from PyQt5 import QtCore, QtGui, QtWidgets
  
  from qweechat import config
  from qweechat.about import AboutDialog
  from qweechat.buffer import BufferListWidget, Buffer
  from qweechat.connection import ConnectionDialog
